packages/snaps-execution-environments/src/common/endowments/commonEndowmentFactory.ava.test.ts

```diff
@@ -0,0 +1,30 @@
+// eslint-disable-next-line import/no-unassigned-import
+import 'ses';
+import test from 'ava';
+// FinalizationRegistry will fix type errors in tests related to network endowment.
+// eslint-disable-next-line import/no-extraneous-dependencies, @typescript-eslint/no-unused-vars
+import FinalizationRegistry from 'globals';
+
+import buildCommonEndowments, {
+  EndowmentFactory,
+} from './commonEndowmentFactory';
+
+// Note: harden is only defined after calling lockdown
+lockdown({
+  domainTaming: 'unsafe',
+  errorTaming: 'unsafe',
+  stackFiltering: 'verbose',
+});
+
+const ExpectedNumberOfEndowments = 28;
+
+test('buildCommonEndowments should return an array of endowment factories', (expect) => {
```

Comment:
endowmentModules.ava.test.ts is asserting all that this file is asserting and more. This file seems redundant.